### PR TITLE
Ensure that certain CLI options are always parsed as arrays

### LINKED ISSUE #154: Array options need to be explicitly configured in yargs declaration
**What version of `electron-installer-debian` are you using?**
1.0.0

When using e.g.
`electron-installer-debian --options.depends "['testDep']"`
it has no effect on the dependencies of the resulting package. Specifying it via a config file works fine, though.
I suspect I am using wrong syntax to specify the expected Array[String]. Documentation does not say anything about the syntax, maybe you can help me with that?

### PR DESCRIPTION
**Pull request by fcastilloec**
Fixes #154

### CODE REVIEW

File: test/cli.js [hunk 1]
```diff
@@ -60,4 +60,28 @@ describe('cli', function () {
...
+      '--dest', outputDir,
+      '--arch', 'amd64',
+      '--options.depends', 'testDepend', 'testDepend2',
+      '--options.recommends', 'testRecommends',
+      '--options.suggests', 'testSuggests',
+      '--options.enhances', 'testEnhances',
+      '--options.preDepends', 'testPreDepends',
+      '--options.categories', 'testCategory', 'testCategory2', 'testCategory3',
+      '--options.mimeType', 'testMimeType',
+      '--options.lintianOverrides', 'testLitianOverride'
```
Comment (malept) [inline test/cli.js]:
```suggestion
      '--options.lintianOverrides', 'testLintianOverride'
```

**Comment by fcastilloec** [comment]
I was also wondering if a test is needed. If you also have reservations, maybe we should just not add one. Thoughts @malept ?

**Comment by malept** [comment]
It might not be worth it, mostly because it's testing the yargs implementation. On the other hand, I like regression tests, so I'm a little conflicted.

**Comment by fcastilloec** [comment]
I've decided to remove it. As you mentioned, this is not testing our code, but rather `yargs` implementation.

**Pull request merged**